wiki/People/Beatmap_Nomination_Group/Rules/en.md

```diff
@@ -36,7 +36,7 @@ Violations of those rules may be reported to a member of the [Quality Assurance
 + **Ensure that kudosu is awarded to all posts that deserve it.** Additionally consult the mapper before giving out kudosu to make sure there are not any reasons why the post does not need kudosu (e.g. the modder has received one already for a different post which you overlooked). If the modpost is too old to receive kudosu, consult the modder to make an additional post linking the original he receives kudosu for.
 + **You must not place any icons besides the star, bubble, bubble pop and heart.** Whilst you are able to set other icons, they are either deprecated and discouraged from use (flame, circle, drum, apple, piano) or reserved for use by the Quality Assurance Team (nuke). If you accidentally set an unwanted icon, contact a QAT immediately or overwrite the icon in case you wanted to post a bubble or heart.
 + **Do not nominate a beatmap without properly checking all difficulties of the beatmap all by yourself.** If you are unfamiliar with the game mode some of the difficulties are mapped in, ask fellow Nominators for help. Improper checking includes just running a modding tool before nominating or not even downloading the beatmap. 
-+ **You cannot rebubble a map you nominated until the Nominator who placed the bubble-pop agrees to it.** In these cases another nominator may rebubble the map if they disagree with the bubble pop.
++ **You must not renominate a map in which your nomination was vetoed, unless the respective vetoing nominator agrees to that action.** When nominating over a veto, you have to exhaustively explain why the reasons for the veto do not hold the map back from reaching Ranked.
```

Comment:
nitpicking
`for the veto do not hold the map` to `for the veto will not hold the map`